### ISSUE TITLE
Ped variations missing 

### ISSUE THREAD

**Issue by Truffleshuffle03** [itsjustcurtis/MenyooSP#162]
Free mode peds have been fixed I saw in a penned comment but there are still a lot of peds missing. For example, scenario male sheriff is missing at least two ped variations. Trucks are missing from the vehicle spawn list but mainly ped variations are my biggest issue that are missing.  I have noticed others besides the sheriff variations missing but since I just started using this menyoo I can't remember them off the top of my head. I have been using Simple Trainer for spawning them but it's a headache switching back and forth since Menyoo is my main trainer.

**Comment by itsjustcurtis** [comment]
if you are able to be more specific with the spawn names of the missing entities then they can be added quite easily. All it takes is an update to the pedlist.xml file

**Comment by Truffleshuffle03** [comment]
> if you are able to be more specific with the spawn names of the missing entities then they can be added quite easily. All it takes is an update to the pedlist.xml file

yes I will try I just have not used this version much yet those are just what I ran into with limited time using it so far.

**Comment by itsjustcurtis** [comment]
https://raw.githubusercontent.com/DurtyFree/gta-v-data-dumps/refs/heads/master/peds.json

Full list of peds for comparison

**Comment by itsjustcurtis** [comment]
Done a quick comparison, looks like there are some 400+ peds missing currently, will take some effort to correct as they are unfortunately hard-coded
